test/functionalTest/cases/0000_ngsild/ngsild_entity_post_operations_upsert.test

```diff
@@ -0,0 +1,243 @@
+        },
+        "engine": {
+            "type": "Property",
+            "value": "V8"
+        },
+        "isParked": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:OffStreetParking:Downtown1",
+            "observedAt": "2017-07-29T12:00:04Z",
+            "providedBy": {
+                "type": "Relationship",
+                "object": "urn:ngsi-ld:Person:Bob"
+            }
+        }
+    },
+    {
+        "id": "urn:ngsi-ld:Vehicle:00003",
+        "type": "Vehicle",
+        "brandName": {
+            "type": "Property",
+            "value": "Mercedes"
+        },
+        "isParked": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:OffStreetParking:Downtown1",
+            "observedAt": "2017-07-29T12:00:04Z",
+            "providedBy": {
+                "type": "Relationship",
+                "object": "urn:ngsi-ld:Person:Bob"
+            }
+        },
+        "isBlocked": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:OffStreetParking:Downtown1",
+            "observedAt": "2017-07-29T12:00:04Z",
+            "providedBy": {
+                "type": "Relationship",
+                "object": "urn:ngsi-ld:Person:Bob"
+            }
+        }
+    },
+    {
+        "id": "urn:ngsi-ld:Vehicle:00004",
+        "type": "Vehicle",
+        "brandName": {
+            "type": "Property",
+            "value": "Mercedes"
+        },
+        "isParked": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:OffStreetParking:Downtown1",
+            "observedAt": "2017-07-29T12:00:04Z",
+            "providedBy": {
+                "type": "Relationship",
+                "object": "urn:ngsi-ld:Person:Bob"
+            }
+        }
+    }
+]'
+orionCurl --url /ngsi-ld/v1/entityOperations/upsert -X POST --payload "$payload" -H "Content-Type: application/ld+json"
+echo
+echo
+
+
+--REGEXPECT--
+01. POST /ngsi-ld/v1/entityOperations/upsert 4 Entities
+=======================================================
+HTTP/1.1 200 OK
+Content-Length: 125
+Content-Type: application/json
+Link: <https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"
+Date: REGEX(.*)
+
+{
+    "success": [
+        "urn:ngsi-ld:Vehicle:00001",
+        "urn:ngsi-ld:Vehicle:00002",
+        "urn:ngsi-ld:Vehicle:00003",
+        "urn:ngsi-ld:Vehicle:00004"
+    ]
+}
+
+
+02. POST /ngsi-ld/v1/entityOperations/upsert Change 2 Entities
+==============================================================
+HTTP/1.1 200 OK
+Content-Length: 69
+Content-Type: application/json
+Link: <https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"
+Date: REGEX(.*)
+
+{
+    "success": [
+        "urn:ngsi-ld:Vehicle:00002",
+        "urn:ngsi-ld:Vehicle:00003"
+    ]
+}
+
+
+--TEARDOWN--
+brokerStop CB
+dbDrop CB
```

Comment:
And of course, we need all kinds of error handling tested.
Everything that can go wrong should be covered my a functest

Reply:
Also, need to check the spec and make sure the service does exactly what it should.

Especially, options=replace, that seems untested so far